### PR TITLE
Booking seat

### PR DESCRIPTION
**Pull request by ConnyPettersson**
This PR contains the page where you choose tickets for Adults, Seniors, or Children, each with different prices. You select tickets by incrementing or decrementing the number of tickets, and the total price is calculated accordingly. Then, you select your seats by marking the white seats, which will turn yellow. You cannot select more seats than you have tickets. Seats that are already booked will be marked in red.

Files added:

Seats/page.tsx
TicketContext.tsx
TicketSelector.tsx
BookingSeat.tsx
Co-writing was done with Albin to export and fetch data from the database.

### CODE REVIEW

**Review by ANschoolacc — APPROVED** [review]
Good job! I really like how you structured the booking page, it looks nice!

**Review by oskber — APPROVED** [review]
Good job! Seems to work well 👍 Remember to switch to single quotes in settings

File: kino-nextjs/app/components/movies/BookingSeat.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          ? row.map((seat, sIndex) =>
+              sIndex === seatIndex
+                ? seat === 0
+                  ? getSelectedSeatsCount() < getTotalSelectedTickets()
+                    ? 2
+                    : seat
+                  : seat === 2
+                  ? 0
+                  : seat
+                : seat
```
Comment (maxlin94) [inline kino-nextjs/app/components/movies/BookingSeat.tsx:38]: Kind of hard (for me at least) to understand what's going on here, but seems to be working :)

**Review by maxlin94 — COMMENTED** [review]
Left some comments, but looks great overall!

**Pull request merged**